### PR TITLE
docs: [PROD-11238] add documentation for dataste manager overview cat…

### PR DESCRIPTION
**Pull request by nborde-CSM**
…egory details

Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

File: doc/datasetManager.md [hunk 1]
```diff
@@ -274,6 +274,131 @@ webApp:
 
 </details>
 
+### Category details
+
+In category accordions, you can open a category details dialog that can display into a Table component, some twingraph dataset data.
+These data are fetched from a Cypher request sent to the twingraph dataset.
+The table is an expression of the twingraph dataset
+The table is read only.
```
Comment (esasova) [inline doc/datasetManager.md:282]: if I'm not mistaken _data_ is uncountable :slightly_smiling_face:  Also, we could simplify this paragraph : 
- the first phrase to _The data is fetched from twingraph dataset with a Cypher query_ 
- not sure we need the second one
:slightly_smiling_face:

File: doc/datasetManager.md [hunk 2]
```diff
@@ -274,6 +274,131 @@ webApp:
...
+The category details dialog can be defined by the key
+`[workspace].webApp.options.datasetManager.categories.previewTable`.
+The value for this field must be an **objects** that contains attributes with the
+following fields:
```
Comment (esasova) [inline doc/datasetManager.md:286]: typo: _object_ (sg).  Also, I think it's to to talk about `keys` and `values` when we define an object :slightly_smiling_face:

File: doc/datasetManager.md [hunk 3]
```diff
@@ -274,6 +274,131 @@ webApp:
...
+  - **nonSortable** (_behavior_)
+- **minValue** (_optional_) minimum value accepted on edition (for column types _int_, _number_ and _date_)\*
+- **maxValue** (_optional_) maximum value accepted on edition (for column types _int_, _number_ and _date_)\*
+- **defaultValue** (_optional_) default value to use when users add new lines; if no default value is defined, the webapp will choose a default value **based on the column type**.
+- **enumValues** (_optional_) list of known enum values, accepted on edition (for column type _enum_)
+- **acceptsEmptyFields** (_optional_) set to `true`, authorizes the column to accept empty fields (default `false`)
+
+For columns of type `date`, when providing values for the options `minValue`, `maxValue` and `defaultValue`,
+the **ISO format** is recommended (these values will be forwarded to the standard JS `Date` constructor).
+
```
[the remark this answers is not in the document]
Comment (nborde-CSM) [reply doc/datasetManager.md:401]: I have found the way to use link and markdown on doc. :+1:

File: doc/datasetManager.md [hunk 4]
```diff
@@ -274,6 +274,131 @@ webApp:
 
 </details>
 
+### Category details
+
+In category accordions, you can open a category details dialog that can display into a Table component, some twingraph dataset data.
```
[the remark this answers is not in the document]
Comment (nborde-CSM) [reply doc/datasetManager.md:279]: I was almost suer this sentence would be incorrect...

File: doc/datasetManager.md [hunk 5]
```diff
@@ -274,6 +274,131 @@ webApp:
...
+                  - int
+                headerName: Number of waiters
+              - field: restockQty
+                type:
+                  - int
+                headerName: Restock quantity
+              - field: stock
+                type:
+                  - int
+                headerName: ' Stock'
```
Comment (csm-thu) [inline doc/datasetManager.md:366]: Same as above, there's an unnecessary space in `' Stock'`

File: doc/datasetManager.md [hunk 6]
```diff
@@ -274,6 +274,131 @@ webApp:
...
+                },
+                {
+                  "field": "restockQty",
+                  "type": ["int"],
+                  "headerName": "Restock quantity"
+                },
+                {
+                  "field": "stock",
+                  "type": ["int"],
+                  "headerName": " Stock"
```
Comment (csm-thu) [inline doc/datasetManager.md:319]: Unnecessary extra space in `" Stock"`

File: doc/datasetManager.md (hunk 1 above)
Comment (nborde-CSM) [reply doc/datasetManager.md:282]: Indeed but it was precision required by the PO.

File: doc/datasetManager.md [hunk 7]
```diff
@@ -274,6 +274,131 @@ webApp:
...
+These data are fetched from a Cypher request sent to the twingraph dataset.
+The table is an expression of the twingraph dataset
+The table is read only.
+The category details dialog can be defined by the key
+`[workspace].webApp.options.datasetManager.categories.previewTable`.
+The value for this field must be an **objects** that contains attributes with the
+following fields:
+
+- `columns`: an array describing **the expected columns of the table** (see section below "Columns definition")
+- `queryId`: the identifier of the _query_ whose results contain the indicator (see section Queries above)
```
[the remark this answers is not in the document]
Comment (nborde-CSM) [reply doc/datasetManager.md:289]: This section has been replaced by a link to the columns-definition in `scenarioParametersConfiguration.md`

File: doc/datasetManager.md [hunk 8]
```diff
@@ -274,37 +274,104 @@ webApp:
 
 </details>
 
-## Dataset creation scripts
+### Category details
 
-### Default transformation scripts
```
Comment (csm-thu) [inline doc/datasetManager.md:279]: Your commit deletes the whole section _Dataset creation scripts / Default transformation scripts_, could you restore it please? :smile:

**Review by csm-thu — CHANGES_REQUESTED** [review]

**Pull request merged**